lerna.json

```diff
@@ -1,5 +1,4 @@
 {
-  "lerna": "2.0.0",
```

Comment:
not required now, apparently?

Reply:
yeah it's not in the docs - i forgot to change it with the upgrade an now complaints